other/buildTools/DCO.txt

```diff
@@ -0,0 +1,25 @@
+Developer's Certificate of Origin 1.1
```

Comment:
Ditto for this -- don't think we need it now, since it's part of the same repo now